loadtest/control/actions.go

```diff
@@ -90,6 +94,7 @@ func Logout(u user.User) UserActionResponse {
 	return UserActionResponse{Info: "logged out"}
 }
 
+// JoinChannel adds the user to a random channel.
```

Comment:
The channel is not randomly picked because we are trying our best to get a successful join.  
We choose the first one which the user is not a member of.

Reply:
by random I mean the channel is selected by chance not with a specific reason. But yes, we can still mention about the logic.

Reply:
We pick the first one that we find that meets our criteria. How the slice of channels is ordered might be considered random but that's up to the server/store to decide and it's probably not.

Also, since we use pseudo random utilities in other places I think it's better to specify the difference with this approach.